seq_logger.js

```diff
@@ -43,12 +40,6 @@ class SeqLogger {
         this._activeShipper = null;
         this._onRemoteConfigChange = cfg.onRemoteConfigChange || null;
         this._lastRemoteConfig = null;
-
-        this._httpModule = this._endpoint.protocol === "https:" ? https : http
-        this._httpAgent = new this._httpModule.Agent({
```

Comment:
the reference to _httpAgent on line 96 causes a crash

Reply:
Removed